Libraries/process/Sources/DefaultLauncher.cpp

```diff
@@ -101,6 +121,24 @@ launch(Filesystem *filesystem, Context const *context)
         return ext::nullopt;
     } else {
         /* Fork succeeded, existing process. */
+        if (pipe_setup_success) {
+            close(pfd[1]);
+            /* Read child's stdout/stderr through pipe, and output stdout */
+            while (true) {
+                char pin;
+                int readlen = read(pfd[0], &pin, 1);
```

Comment:
Yeah, I guess I was being lazy and reading one char at a time.
Given that this will try to read many bytes of output from child processes, it makes sense to buffer them.